packages/react/src/components/form/Textfield/Textfield.mdx

```diff
@@ -54,30 +52,37 @@ Bruk `counter` til å begrense antall tegn brukerne kan skrive i feltet.
 
 ## Retningslinjer for når du skal bruke `Textfield`
 
-Vi bruker tekstfelt når vi vil gi brukeren mulighet til å skrive tekst/svar på maks. én linje, Det kan for eksempel være navn eller telefonnummer.
+Vi bruker `textfield` når vi vil gi brukeren mulighet til å skrive tekst eller korte svar på maksimalt én linje. Dette kan for eksempel være navn eller telefonnummer.
 
-Passer til å
+<br />
 
-- gi mulighet for korte tekster eller svar
-- legge inn tall, for eksempel et telefonnummer
+### Passer til
 
-Passer ikke til å
+- å gi mulighet for korte tekster eller svar.
+- å legge inn tall, for eksempel et telefonnummer.
+
+### Passer ikke til 
+
+- å gi lengre svar, bruk heller `textarea`.
+- å legge inn formaterte data, som markdown.
```

Comment:
Denne virka litt rar for meg, `Textfield` inneheld **både** `input` og `textarea`.. kor skal du då skrive lengre tekstar?